modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -8,62 +8,50 @@
 
 After you install the NUMA Resources Operator, do the following to deploy the NUMA-aware secondary pod scheduler:
 
```

Comment:
@ffromani - I removed the bullet here saying you need to create a MCP and a pod admittance policy. 

I think the pod admittance policy is not needed. And the MCP was created during the procedure to create the NUMAResourcesOperator custom resource. So instead, I moved the task to create an MCP as a prerequisite.

Reply:
"pod admittance policy" is one of the things we can configure using a performance profile. I think the latest version you uploaded is better.